scripts/releaseTag.js

```diff
@@ -0,0 +1,92 @@
+const childProcess = require('child_process');
+const fse = require('fs-extra');
+const path = require('path');
+const { promisify } = require('util');
+const yargs = require('yargs');
+
+/**
+ * Only directly call it with side-effect free commands.
+ * Otherwise use the `exec` that's considering whether the context is supposed to be "dry" i.e. have no side-effects.
+ */
+const execActual = promisify(childProcess.exec);
+/**
+ * @param {string} command
+ * @param {unknown} [options]
+ */
+function execDry(command, options) {
+  // eslint-disable-next-line no-console
+  console.log(`exec(\`${command}\`, ${JSON.stringify(options)})`);
+}
+
+/**
+ * Find the remote pointing to mui/mui-x.
+ *
+ * Conventionally this should be named `upstream` but some collaborators might've used a different naming scheme.
+ */
+async function findMuiXRemote() {
+  const { stdout } = await execActual(['git', 'remote', '-v'].join(' '));
+  const remoteLines = stdout.trim().split(/\r?\n/);
+
+  return remoteLines
+    .map((remoteLine) => {
+      const [name, url, method] = remoteLine.split(/\s/);
+      return { name, url, method };
+    })
+    .find((remote) => {
+      // test: https://regex101.com/r/fBVJUX/1
```

Comment:
I first wonder why you run test on such a strange url 😅
```suggestion
      // To test the regexp, go to: https://regex101.com/r/fBVJUX/1
```